### DISCUSSION TITLE
Unable to properly control my fans

### DISCUSSION THREAD

**Discussion by Torduu** [T-Troll/alienfx-tools#366] [Q&A]
Hello :) 
First of all: thank you soooooo much for this software. Finally something simple and efficient. I love your style ;)
My issue is with fan control. I can't find what i'm doing wrong :( and it has high chances to be a user problem ._.
My fans run at 68% even when I try to stop them.
<img width="697" alt="AlienFx Fan curve" src="https://github.com/T-Troll/alienfx-tools/assets/142119399/dc83d717-2d44-43dd-b0db-414e5fad69de">
P.S. The GPU fan curve is identical to CPU and both are linked to cpu internal thermistor.
The system is a m15 r4, I7 10870h, rtx 3070 mobile, and I use MSI afterburner and throttle stop. ( I would love to uninstall AWCC XD)
I can increase fans all the way to 100% and i did check max boost but i can't go lower than 68%
Any hint will be very appreciated. thank you

**Comment by T-Troll** [comment]
As well as AWCC, fan control is indirect. Boost applied on top of BIOS settings (and this is for good).
So 68% at 0 boost is BIOS setting.
BTW, 68% a bit high for 38C, can be BIOS bug...

**Comment by Torduu** [comment]
In AWCC when I use a custom fan curve my fans runs at 68% at their lowest and when I switch to balance profile they are at 0% until cpu reach 69c.
<img width="976" alt="AWCC balanced Fan" src="https://github.com/T-Troll/alienfx-tools/assets/142119399/255ec8b8-b305-46a2-8931-fdd4e8b4b0fa">
<img width="488" alt="AWCC Fan curve" src="https://github.com/T-Troll/alienfx-tools/assets/142119399/a5beed2f-03eb-4311-9f49-b39252eb42fd">

fans can run at 36% so Maybe it is BIOS or could it be a driver? I'm not really knowledgeable about driver and bios.

Thank you for your the quick reply, also I realised, after posting, that I posted in the wrong section, sorry about that.

**Reply by T-Troll** [reply]
> fans can run at 36% so Maybe it is BIOS

BIOS. BTW, you can switch Power mode into my app as well (it's the same as into AWCC Thermal mode). But fan control only avaliable into manual mode.
You can also switch it by keyboard shortcut - CTRL+ALT+0...9 (0 - manual).

**Reply by Torduu** [reply]
yeah I found out that level 163 seems to be balance or cool profile so I can use your app to do the same thing as AWCC but I would prefer to have my fans start at 55c 35% and 100% at 80c in manual mode

**Reply by T-Troll** [reply]
Up to you, i glad to provide tool to do so.
NB: You can rename power mode - just type a new name into it's edit box. Manual can't be renamed.

**Reply by Torduu** [reply]
yes i can rename, but I don't understand why the software can't run fan at 35% on manual.

**Reply by T-Troll** [reply]
Well... Being short, the BIOS have different fan tables for each mode. Manual tuned like this (as well as other settings).
Ask Dell, please, not me...

**Reply by Torduu** [reply]
ok, thanks for your help and the wonderful job you did with the software :) have a good day
